pkg/testrunner/template/template.go

```diff
@@ -153,6 +160,7 @@ func renderChartWithShoot(log logr.Logger, renderer *templateRenderer, parameter
 				"floatingPoolName":       shoot.FloatingPoolName,
 				"loadbalancerProvider":   shoot.LoadbalancerProvider,
 				"infrastructureConfig":   infrastructure,
+				"networking":             networking,
```

Comment:
lets name it `networkingConfig` to have the same naming as for the other configurations